NosTests/Test Helpers/Bundle+Localization.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+// This is some AI generated code that allows us to set a specific locale in unit tests so we can do things like 
+// assert on English strings no matter what locale the simulator/host is set to.
```

Comment:
I think I've accomplished this a different way before, but I'll have to try to remember how I did it. I don't have a problem with this approach if it works, but it sounds like it's failing for @martindsq.